src/components/Reports/LocationSection.tsx

```diff
@@ -355,6 +417,108 @@ export default function LocationSection(props: LocationSectionProps): JSX.Elemen
+                  isEditable={false}
+                  title={strings.CURRENT_NEXT_OBSERVATION}
+                  contents={`${currentObservation ? currentObservation?.startDate : ''} - ${
+                    nextObservation ? nextObservation?.startDate : ''
+                  }`}
+                  className={classes.infoCardStyle}
+                />
+              </Grid>
+              <Grid item xs={smallItemGridWidth()}>
+                <OverviewItemCard
+                  isEditable={false}
+                  title={strings.TOTAL_PLANTS_OBSERVED}
+                  contents={latestObservation.totalPlants}
+                  className={classes.infoCardStyle}
+                />
+              </Grid>
+              <Grid item xs={smallItemGridWidth()}>
+                <OverviewItemCard
+                  isEditable={false}
+                  title={strings.LIVE_PLANTS_OBSERVED}
+                  contents={getLivePlants() || ''}
+                  className={classes.infoCardStyle}
+                />
+              </Grid>
+              <Grid item xs={smallItemGridWidth()}>
+                <OverviewItemCard
+                  isEditable={false}
+                  title={strings.DEAD_PLANTS_OBSERVED}
+                  contents={getDeadPlants() || ''}
+                  className={classes.infoCardStyle}
+                />
+              </Grid>
+              <Grid item xs={smallItemGridWidth()}>
+                <OverviewItemCard
+                  isEditable={false}
+                  title={strings.SPECIES_OBSERVED}
+                  contents={latestObservation.totalSpecies}
+                  className={classes.infoCardStyle}
+                />
+              </Grid>
+              <Grid item xs={smallItemGridWidth()}>
+                <OverviewItemCard
+                  isEditable={false}
+                  title={strings.MARKED_AS_PLANTING_COMPLETE}
+                  contents={getMarkedAsComplete()}
+                  className={classes.infoCardStyle}
+                />
+              </Grid>
+              <Grid item xs={smallItemGridWidth()}>
+                <OverviewItemCard
+                  isEditable={false}
+                  title={strings.MORTALITY_RATE_PERCENT}
+                  contents={latestObservation.mortalityRate}
+                  className={classes.infoCardStyle}
+                />
+              </Grid>
+              <Grid item xs={smallItemGridWidth()}>
+                <OverviewItemCard
+                  isEditable={false}
+                  title={strings.PLANTING_PROGRESS_PERCENT}
+                  contents={reportedPlants?.progressPercent || ''}
+                  className={classes.infoCardStyle}
+                />
+              </Grid>
+              <Grid item xs={smallItemGridWidth()}>
+                <OverviewItemCard
+                  isEditable={false}
+                  title={strings.PLANTING_DENSITY_OF_PLANTED_ZONES}
+                  contents={''}
```

Comment:
Missing this value, not sure what we want to display

Reply:
Observation results payload has planting progress and planting density. Not sure what `reportedPlants` is in this change. Should it be the observation results instead?

Reply:
I don't see the plantingProgress in the observationResults payload, that's why I got it from reportedPlants, as we do in the PlantingSiteProgressCard in the dashboard

Reply:
And for density we want to show the value per zone, so I'm adding same logic as we do in the PlantingDensityPerZoneCard

Reply:
> And for density we want to show the value per zone, so I'm adding same logic as we do in the PlantingDensityPerZoneCard

Possible I don't understand the requirements. Density will be different for different zones. The density per zone card shows barchart for various zones. Here it is just one number, for the most recent observation result perhaps. Not sure it's the same as planting density per zone use-case? There's a density for the observation result, which is populated only when all subzones are marked as completed - if that's what you want.

Reply:
> I don't see the plantingProgress in the observationResults payload, that's why I got it from reportedPlants, as we do in the PlantingSiteProgressCard in the dashboard

Got it, I see it in the generated types for reported plants api.

Reply:
> > And for density we want to show the value per zone, so I'm adding same logic as we do in the PlantingDensityPerZoneCard
> 
> Possible I don't understand the requirements. Density will be different for different zones. The density per zone card shows barchart for various zones. Here it is just one number, for the most recent observation result perhaps. Not sure it's the same as planting density per zone use-case? There's a density for the observation result, which is populated only when all subzones are marked as completed - if that's what you want.

This is not one value, is a list of <zone: value> (you can check in design) . Going to update the code

Reply:
Ah ok got it. I didn't see the designs. Update the screenshot here if you can?